CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+## 2.11.1
```

Comment:
It possibly could go out with `2.11.0` depending on the release schedule (it's not out yet)

Reply:
Good point and done! Thank you @marcbouchenoire